src/Configuration/ConfigurationGraphViz.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Qossmic\Deptrac\Configuration;
+
+final class ConfigurationGraphViz
+{
+    /** @var array<string, string[]> */
+    private $groupsLayerMap;
+    /** @var string[] */
+    private $hiddenLayers;
+
+    /**
+     * @param array<string, mixed> $arr
+     */
+    public static function fromArray(array $arr): self
+    {
+        return new self($arr['hidden_layers'] ?? [], $arr['groups'] ?? []);
+    }
+
+    /**
+     * @param string[]                $hiddenLayers
```

Comment:
We usually don't align docblocks, because it creates noise in future pull requests.

Reply:
This is done by the cs-fixer configuration. It will not pass tests otherwise. But I agree with your sentiment.